src/newItinerary/internals/ItineraryItem.style.tsx

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components'
+
+import { color } from '../../_utils/branding'
+
+export const StyledItineraryItem = styled.li`
+  min-height: 32px;
+
+  > div,
+  > a {
+    display: flex;
+  }
+
+  a {
+    -webkit-tap-highlight-color: ${color.tapHighlight};
+    -webkit-touch-callout: none;
```

Comment:
It might be a good time ro reassess the usefulness of these properties

Reply:
@Tuxosaurus, do you have any idea?

Reply:
No, but caniuse is pretty clear about that:
https://caniuse.com/?search=callout

It may be worth keeping tap-highlight though:
https://caniuse.com/?search=tap-highlight-color

Looks like it's used on all Items

Reply:
This is what I found on the subject.

[**`-webkit-tap-highlight-color`**](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-tap-highlight-color)

> [6.2 Disable highlighting](https://medium.com/appscope/designing-native-like-progressive-web-apps-for-ios-1b3cdda1d0e8)
>
> ![1_wnXeYxpcqK6KwgfVTNRDKQ](https://user-images.githubusercontent.com/17502801/107778230-1d32bc00-6d44-11eb-9c41-a13eccb26a2b.gif)
>
> When tapping a link in the iOS web browser, a grey box appears around the element. Although there is no simple way of disabling this effect, you may change the highlighting color to transparent, and thus effectively make it disappear. To do this for your PWA, set the `-webkit-tap-highlight-color` property to `transparent` for the desired elements (or assign it to the body element to disable link highlighting for all elements.)

Basically it's a `:active` for iOS Safari.

[**`-webkit-touch-callout`**](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-touch-callout)

> [6.3 Disable callouts](https://medium.com/appscope/designing-native-like-progressive-web-apps-for-ios-1b3cdda1d0e8)
>
> ![image](https://user-images.githubusercontent.com/17502801/107777769-92ea5800-6d43-11eb-8b60-b9274feea83a.png)
>
> If you tap and hold on an element in the iOS browser, it will open a callout menu (like the one above.) To disable this effect on iOS, set the `-webkit-touch-callout` property to `none` for the desired elements. Again, to disable the effect for all elements, assign the property to your body element.

Reply:
As always, thanks @simonrelet for your explanations! 
Based on those, I think we can get rid of `-webkit-touch-callout` as it goes against user's habits. I think we can leave the other one though.